cmd/cli/command/availability_spec/delete.go

```diff
@@ -0,0 +1,40 @@
+package availabilityspec
+
+import (
+	"fmt"
+
+	"connectrpc.com/connect"
+	"github.com/common-fate/sdk/config"
+	configv1alpha1 "github.com/common-fate/sdk/gen/commonfate/control/config/v1alpha1"
+	"github.com/common-fate/sdk/service/control/configsvc"
+	"github.com/urfave/cli/v2"
+)
+
+var deleteCommand = cli.Command{
+	Name:  "delete",
+	Usage: "Delete an Availability Spec",
+	Flags: []cli.Flag{
+		&cli.StringFlag{Name: "availability-spec-id", Required: true, Usage: "the Availability Spec ID to delete"},
```

Comment:
Maybe just opt for the shorter "id" here for brevity, since the command is already `cf availability-spec delete --id` 
```suggestion
		&cli.StringFlag{Name: "id", Required: true, Usage: "the Availability Spec ID to delete"},
```